ztp/extra-manifests-builder/99-sync-time-once/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+SYNC_ATTEMPT_TIMEOUT_SEC=${SYNC_ATTEMPT_TIMEOUT_SEC:-300}
+
+echo "\
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${MCPROLE}
+  name: 99-sync-time-once-${MCPROLE}
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Sync time once
+            After=network.service
+            [Service]
+            Type=oneshot
+            TimeoutStartSec=${SYNC_ATTEMPT_TIMEOUT_SEC}
```

Comment:
This is even cleaner - Let chronyd run as long as it likes, and leave the termination up to systemd.  I like it!